cdhweb/people/models.py

```diff
@@ -495,3 +498,9 @@ def init_person_from_ldap(user, ldapinfo):
     # always update PU status to current
     person.pu_status = str(ldapinfo.pustatus)
     person.save()
+
+
+class PersonRelatedLink(Orderable, RelatedLink):
+    '''Through-model for associating people with resource types and
+    corresponding URLs for the specified resource type.'''
+    person = ParentalKey(Person, on_delete=models.CASCADE, related_name="related_links")
```

Comment:
I've belatedly realized that we don't want `Orderable` here (although having it does nothing). That mixin adds a `sort_order` field, but we already define that field manually on `RelatedLinkType`, and we want it to be editable in the admin so that links are always ordered based on their type (rather than reorderable on each page). So, you can just remove the references to `Orderable` for `RelatedLink` subtypes (and theoretically nothing should happen).